stix_shifter_utils/stix_translation/src/json_to_stix/json_to_stix_translator.py

```diff
@@ -1,41 +1,44 @@
 import re
 import uuid
+import json
 
 from stix_shifter_utils.stix_translation.src.json_to_stix import observable
 from stix2validator import validate_instance, print_results
 from datetime import datetime
 from stix_shifter_utils.utils import logger
 
+# "ID Contributing Properties" taken from https://docs.oasis-open.org/cti/stix/v2.1/csprd01/stix-v2.1-csprd01.html#_Toc16070594
+UUID5_NAMESPACE = "00abedb4-aa42-466c-9c01-fed23315a9b7"
+NUMBER_OBSERVED_KEY = 'number_observed'
+FIRST_OBSERVED_KEY = 'first_observed'
+LAST_OBSERVED_KEY = 'last_observed'
 
-# convert JSON data to STIX object using map_data and transformers
 
+# convert JSON data to STIX object using map_data and transformers
 def convert_to_stix(data_source, map_data, data, transformers, options, callback=None):
-    bundle = {
-        "type": "bundle",
-        "id": "bundle--" + str(uuid.uuid4()),
-        "spec_version": "2.0",
-        "objects": []
-    }
 
-    identity_id = data_source['id']
-    bundle['objects'] += [data_source]
-    data_source_name = data_source['name']
-
-    ds2stix = DataSourceObjToStixObj(identity_id, map_data, transformers, options, data_source_name, callback)
+    ds2stix = DataSourceObjToStixObj(data_source, map_data, transformers, options, callback)
 
     # map data list to list of transformed objects
-    results = list(map(ds2stix.transform, data))
+    observation = ds2stix.transform
+    results = list(map(observation, data))
+
+    for object in results:
```

Comment:
I wonder if its safe to use `object` as variable name. it usually a keyword in python

Reply:
I'll change it just to be safe.